lib/Dialect/Triton/IR/Ops.cpp

```diff
@@ -969,12 +969,18 @@ OpFoldResult AdvanceOp::fold(FoldAdaptor adaptor) {
 //-- MakeTensorDescOp --
 void MakeTensorDescOp::build(OpBuilder &builder, OperationState &state,
                              Value base, ValueRange shape, ValueRange strides,
-                             ArrayRef<int32_t> blockShape) {
+                             ArrayRef<int32_t> blockShape,
+                             bool isSignedInteger) {
   auto ptrTy = dyn_cast<triton::PointerType>(base.getType());
   if (!ptrTy) {
     llvm::report_fatal_error("Expected pointer type");
   }
   auto elemTy = ptrTy.getPointeeType();
+  if (isSignedInteger) {
+    auto intTy = cast<IntegerType>(elemTy);
```

Comment:
Should this also make `IntegerType::Unsigned`?

Reply:
Ah good catch, changed to call the isSigned builder instead.